resources/views/partials/hero-simple.blade.php

```diff
@@ -0,0 +1,20 @@
+<!-- start hero -->
+<div class="hero relative z-10 mt-[-125px] w-full overflow-hidden pt-[125px]">
+    <img
+        src="/img/rapidez-bg.jpg"
+        class="absolute inset-0 h-full w-full"
+    ></img>
```

Comment:
```suggestion
    <img src="/img/rapidez-bg.jpg" class="absolute inset-0 h-full w-full"/>
```
Image tags don't need a closing tag and 1 line is enough